repo/migrations.go

```diff
@@ -76,7 +77,7 @@ func version(repoPath string) (int, error) {
 	} else if err != nil && os.IsNotExist(err) {
 		version = []byte("0")
 	}
-	v, err := strconv.Atoi(string(version[0]))
+	v, err := strconv.Atoi(string(version[:]))
```

Comment:
This is potentially sig change. Since we're up to double-digits migrations, this was causing some unexpected migration issues (kept moving us back to 1).

Reply:
First time we've had double digit repover, so i doubt we've seen issues before.

Reply:
I meant it was cause _me_ unexpected migration issues because I'm in the dds ;)